components/ui/header.tsx

```diff
@@ -60,23 +63,49 @@ const Nav = () => (
   <Box className="translate-y-6">
     <Flex gap={4}>
       <Link href={"/courses"}>
-        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Courses</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">
+          Courses
+        </Button>
       </Link>
       <Link href={"/tools"}>
-        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Tools</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">
```

Comment:
Not sure is `shadow` props is doing anything here. Doesn't a button already have this style applied in `theme.ts`

Reply:
yeah it is the base button theme I removed the shadow